agent/engine/docker_task_engine_test.go

```diff
@@ -156,6 +156,7 @@ func TestBatchContainerHappyPath(t *testing.T) {
 	default:
 	}
 	eventsReported.Wait()
+	client.EXPECT().DescribeContainer(gomock.Any()).AnyTimes()
```

Comment:
AnyTimes is needed because of following code in concurrent static call from (engine.managedTask).overseeTask 
In another word, if it is taking longer to get to "steady state", the 'DescribeContainer()' will get invoked.  Otherwise, it is NOT invoked.

130                 // If it's steadyState, just spin until we need to do work
131                 for mtask.steadyState() {
132                         llog.Info("Task at steady state", "state", mtask.KnownStatus.String())
133                         maxWait := make(chan bool, 1)
134                         timer := mtask.time().After(steadyStateTaskVerifyInterval)
135                         go func() {
136                                 <-timer
137                                 maxWait <- true
138                         }()
139                         timedOut := mtask.waitEvent(maxWait)
140         
141                         if timedOut {
142                                 llog.Debug("Checking task to make sure it's still at steadystate")
143                                 go mtask.engine.CheckTaskState(mtask.Task)
144                         }
145                 }